### ISSUE TITLE
[Bug]: Can't launch from Windows command line without logging to screen and blocking behavior.

### ISSUE THREAD

**Issue by bengalih** [httptoolkit/httptoolkit#619]
### Has this been reported before?

- [X] I have checked for existing reports of this issue

### Repro steps

If launching on windows by clicking on "HTTP Toolkit.exe", or by entering the full path to the exe in Start > Run, the application launches as normally with only the one HTTP Toolkit window.

However, if trying to script the launching in any fashion, even as simply as launching it from a command prompt, the following occurs:

```
C:\Users\bengalih>"C:\Users\bengalih\AppData\Local\Programs\httptoolkit\HTTP Toolkit.exe"

C:\Users\bengalih>
Config checked in 36 ms
Certificates setup in 10 ms
Standalone server started in 10 ms
Server started in 34 ms
Total startup took 90 ms
(node:25284) Warning: Using insecure HTTP parsing
(Use `node --trace-warnings ...` to show where the warning was created)
Mock session started, http on port 8000, webrtc enabled
Created docker-tunnel-proxy-8000
Docker not available: connect ENOENT //./pipe/docker_engine
Detected terminal command: C:/Program Files/git/git-bash.exe
Webextension installed at C:\Users\bengalih\AppData\Local\Temp\httptoolkit-webextension
Usbmux iOS scanning connected
httptoolkit-server: Updating CLI... fetching manifest
httptoolkit-server: Updating CLI... already on latest version: 1.18.1
(node:25284) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Browser cache updated
httptoolkit-server: Updating CLI... fetching manifest
httptoolkit-server: Updating CLI... already on latest version: 1.18.1
```

And the command prompt becomes locked up until closing the app, whereupon control returns back to the prompt (i.e. you can type more).

This prevents any type of scripting which includes launching the app.

Why is this happening and how to prevent this?

### How often does this bug happen?

Every time

### The desktop OS you're using

Windows 11

**Comment by pimterry** [comment]
This is all correct and working as expected. The messages shown there are normal output from HTTP Toolkit startup. When you launch most programs from a command line, your terminal will keep running them in the foreground (so your input goes to the program, not your terminal) until they exit. This is standard behaviour that's not related to HTTP Toolkit.

If you want to launch HTTP Toolkit in the background, you'll need to use command line constructs and commands to do so, just like launching any other process. https://stackoverflow.com/questions/21031171/how-to-run-a-command-in-the-background-on-windows has more info on this.

**Comment by bengalih** [comment]
This is not expected behavior on Windows, and for non native windows apps, some of the methods that are outlined in the stackflow article work (though most miss the mark as to what I am trying to solve).

To be clear, almost any native windows application will demonstrate the same behavior whether you click on its executable to run it, or if you type the path to it's executable in a terminal window (cmd or ps).  An exception might be say, a script such as a .py program.

If I write a simple hello world and run it:
```
C:\Users\bengalih\Desktop>hello.py
Hello, World!

C:\Users\bengalih\Desktop>start "hello" hello.py
```
Note running it standard will display the output in the existing terminal window.  However, using a mechanism like "start" (one I tried and one listed in the article you linked), will cause the hello.py to be executed in its own window with no further execution in the existing one (not the window will flash and close almost immediately).

This is not the behavior that http-toolkit exhibits, it will still log everything I have shown above even when using this syntax.
I realize this is not a native windows application, and don't fully understand how it has been ported over and wrapped to run on windows.  However, it exhibits behavior that is different than any other application or script I have ever dealt with on windows exhibits.

What is interesting is that if you launch it in a cmd window, it doesn't appear to block additional execution, but it will continue to log to the current window injecting logging code into whatever is showing in the existing terminal.

I did find a way to bypass this, that I have not had to go to these extents with any other program and that is to open up an additional cmd instance which immediately terminates:

`cmd /c start "HTTP Toolkit" "C:\Users\bhendin\AppData\Local\Programs\httptoolkit\HTTP Toolkit.exe"`

Maybe this will help someone else who is experiencing the same strangeness.
